src/components/Expander/Expander.spec.jsx

```diff
@@ -0,0 +1,134 @@
+				assert.equal(shallow(<Expander isExpanded={true} />).find(ChevronIcon).prop('direction'), 'up');
+			});
+
+			it('sets the value of the `direction` prop on its `ChevronIcon` instance to "down" when `false`.', () => {
+				assert.equal(shallow(<Expander isExpanded={false} />).find(ChevronIcon).prop('direction'), 'down');
+			});
+
+			it('adds the "lucid-Expander-is-expanded" class to the root element when `true`.', () => {
+				assert.equal(shallow(<Expander isExpanded={true} />).find('.lucid-Expander-is-expanded').length, 1);
+			});
+
+			it('adds the "lucid-Expander-content-is-expanded" class to its content container element when `true`.', () => {
+				assert.equal(shallow(<Expander isExpanded={true} />).find('.lucid-Expander-content-is-expanded').length, 1);
+			});
+
+			it('defaults to `false`.', () => {
+				assert.equal(mount(<Expander />).prop('isExpanded'), false);
+			});
+		});
+
+		describe('onExpand', () => {
+			it('defaults to the Lodash `noop` method.', () => {
+				assert.equal(mount(<Expander />).prop('onExpand'), _.noop);
+			});
+		});
+
+		describe('Label (as a prop)', () => {
+			it('renders the value in the header in a `SPAN` element neighboring its `ChevronIcon` instance.', () => {
+				assert.equal(
+					shallow(<Expander Label='foo' />)
+							.find(ReactCSSTransitionGroup)
+							.find('span')
+							.prop('children'),
+					'foo'
+				);
+			});
+		});
+
+		describe('Label (as a child)', () => {
+			it('renders the value in the header in a `SPAN` element neighboring its `ChevronIcon` instance.', () => {
+				const wrapper = shallow(
+					<Expander>
+						<Expander.Label>foo</Expander.Label>
+					</Expander>
+				);
+
+				assert.equal(
+					wrapper
+							.find(ReactCSSTransitionGroup)
+							.find('span')
+							.prop('children'),
+					'foo'
+				);
+			});
+		});
+
+		describe('pass throughs', () => {
+			it('passes through all props not defined in `propTypes` to the root element.', () => {
+				const wrapper = shallow(
+					<Expander
+							className='wut'
+							isExpanded={true}
+							onExpand={_.noop}
+							style={{ marginRight: 10 }}
+							foo={1}
+							bar={2}
+							baz={3}
+							qux={4}
+							quux={5}
+					/>
+				);
+				const rootProps = wrapper.find('.lucid-Expander').props();
+
+				// It should pass `foo`, `bar`, `baz`, `qux`, and `quux` through
+				// to the `Switch` instance.
+				_.forEach(['foo', 'bar', 'baz', 'qux', 'quux'], (prop) => {
+					assert(_.has(rootProps, prop));
+				});
+			});
+		});
+	});
+});
+
+describeWithDOM('Expander', () => {
+	describe('user clicks on the header', () => {
+		it('calls the function passed in as the `onExpand` prop...', () => {
+			_.forEach(['-header', '-icon', '-text'], (classSubstring) => {
+				const onExpand = sinon.spy();
+
+				mount(<Expander onExpand={onExpand} />).find(`.lucid-Expander${classSubstring}`).simulate('click');
```

Comment:
as a good practice you should call `.unmount()` on these wrappers which use the DOM. I might not be necessary in your case, but some components do cleanup tasks in the DOM during the `componentWillUnmount` lifecycle method, which won't be invoked if you don't call `.unmount()`.

For Example, anything which uses Portal depends on this this lifecycle method to cleanup the DOM for the next test. Again, might not be necessary for this component, but it's a good habit to call `.unmount()` in an `afterEach` section :)